src/essreflectometry/supermirror/types.py

```diff
@@ -2,9 +2,12 @@
 
 import scipp as sc
 
-# TODO What do they mean?
-# Supermirror parameters
+# TODO Better specification of what the supermirror parameters mean
```

Comment:
Why not do this now?

Reply:
Because I don't know the definitions

Reply:
This change was just made to make the parameters visible in the docs